test/new/db/cmd/cmd_open

```diff
@@ -260,30 +260,29 @@ isq~?
 EOF
 RUN
 
-# TODO: re-enable when r2r.js and v are dead
-#NAME=ob 0 fix
-#FILE=<<EOF
-#-
-#-
-#-
-#EOF
-#CMDS=<<EOF
-#ob~[0-1]
-#?e
-#ob 1; i~^fd
-#ob 2; i~^fd
-#ob 0; i~^fd
-#EOF
-#EXPECT=<<EOF
-#0 3
-#1 4
-#2 5
-#
-#fd       4
-#fd       5
-#fd       3
-#EOF
-#RUN
+NAME=ob 0 fix
+FILE=<<EOF
+-
```

Comment:
this doesnt looks good at all

Reply:
ok fixed 59a7711574e1e434d2e380d9d52f5f0b1394d244